deploy/criReturnStepFunction.asl.json

```diff
@@ -35,26 +35,8 @@
       "ResultPath": "$.lambdaResult",
       "Next": "RouteJourneyResponse"
     },
-    "ProcessJourneyStep": {
-      "Type": "Task",
-      "Resource": "${IPVProcessJourneyStepFunctionArn}",
-      "Parameters": {
-        "ipvSessionId.$": "$.ipvSessionId",
-        "ipAddress.$": "$.ipAddress",
-        "journey.$": "$.lambdaResult.journey"
-      },
-      "ResultPath": "$.lambdaResult",
-      "Next": "RouteJourneyResponse"
-    },
-    "ProcessJourneyStep-JourneyError": {
-      "Type": "Task",
-      "Resource": "${IPVProcessJourneyStepFunctionArn}",
-      "Parameters": {
-        "ipvSessionId.$": "$.ipvSessionId",
-        "ipAddress.$": "$.ipAddress",
-        "journey": "journey/error"
-      },
-      "ResultPath": "$.lambdaResult",
+    "RetrieveCriOAuthAccessTokenError": {
+      "Type": "Pass",
```

Comment:
This should pass the following to the next step, rather than the contents of the previous:
```json
{ "journey": "/journey/error" }
```

e.g.
```
"RetrieveCriOAuthAccessTokenError": {
  "Type": "Pass",
  "Result": {
    "journey": "/journey/error" 
  },
  "Next": "ReturnToFrontend"
}
```

Reply:
Done